### PR TITLE
Fix I8I8I SDWA Fail for gfx12

### PR DESCRIPTION
**Pull request by wenchuanchen**
gfx12 removes sdwa features, sdwa=None results in data compared fail.
The sdwa usage: https://llvm.org/docs/AMDGPUModifierSyntax.html#sdwa-modifiers

### CODE REVIEW

**Review by cmingch — DISMISSED** [review]
lgtm

**Comment by wenchuanchen** [comment]
fix python syntax warning, replace "is" with "==" for MAC_F32.py and MAC_F64.py.

File: tensilelite/Tensile/Components/MAC_F32.py
Comment (cmingch) [inline tensilelite/Tensile/Components/MAC_F32.py:1]: it's not related to the commit description

File: tensilelite/Tensile/Components/MAC_F64.py
Comment (cmingch) [inline tensilelite/Tensile/Components/MAC_F64.py:1]: it's not related to the commit description

File: tensilelite/Tensile/Tests/common/gemm/gfx12/i8i8i_test_sdwa.yaml
Comment (cmingch) [inline tensilelite/Tensile/Tests/common/gemm/gfx12/i8i8i_test_sdwa.yaml:1]: yaml for testing don't need to reply on what the issue you resolve. what I mean is naming to sdwa is not required. btw, you can just use original i8 test case to cover i8i8i instead creating new one

**Review by cmingch — APPROVED** [review]

**Review by hcman2 — APPROVED** [review]
ok if ci passed

**Pull request merged**